substrate/frame/support/test/tests/pallet_ui/call_feeless_invalid_type.stderr

```diff
@@ -0,0 +1,5 @@
+error: expected `|`
+  --> tests/pallet_ui/call_feeless_invalid_type.rs:31:24
+   |
+31 |         #[pallet::feeless_if(0)]
+   |                              ^
```

Comment:
Not sure why this doesn't say something like 'expected closure' or 'expected ||' instead of just a single '|'

Reply:
Done